Update dev_env_mac.md (#2239)
PX4/PX4-user_guide: en/dev_setup/dev_env_mac.md

## Enable more open files (Handle "LD: too many open files" error)

> Create the `~/.zshenv` file and add insert the line `ulimit -S -n 2048` into it:

Reviewer: ```suggestion
Append the line `ulimit -S -n 2048` into the `~/.zshenv` file (creating it if necessary):
```

```sh

> echo ulimit -S -n 2048 > ~/.zshenv

Reviewer: Thanks, but won't this overwrite the file if it already exists? That is undesirable if you already use the file. Isn't this safer? 

```suggestion
echo ulimit -S -n 2048 >> ~/.zshenv
```

Note, I am not sure whether this will work if the file does not already exist. Can you test it?

Reviewer: If this does not work then we should work out why `open ~/.zshenv` is not working for you. It should.

Reviewer: @julianoes You got much experience on zshenv? I think that the suggested command is a bit dangerous and that `open ~/.zshenv` should work. But would the suggestion I made in https://github.com/PX4/PX4-user_guide/pull/2239#discussion_r1086173273 work?

If not, anyone else we can ask?

Reviewer 2: Yes, append (or create) using `>>` seems correct to me.

Reviewer: Thanks very much!
